internal/cmd/organization/command_list.go

```diff
@@ -0,0 +1,45 @@
+package organization
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type listOut struct {
```

Comment:
I think it would be helpful if we also showed `IsCurrent` for describe and update. It's always going to be true, but just for the user to sanity check what they're doing, and to match other commands.